frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseSlideOutAction.java

```diff
@@ -171,20 +174,25 @@ private void slideOutBranch(AnActionEvent anActionEvent, String branchName) {
     LOG.info("Sliding out '${branchName}' branch in memory");
     val newBranchLayout = branchLayout.slideOut(branchName);
 
-    try {
-      val macheteFilePath = gitRepository.getMacheteFilePath();
-      LOG.info("Writing new branch layout into ${macheteFilePath}");
-      branchLayoutWriter.write(macheteFilePath, newBranchLayout, /* backupOldLayout */ true);
-
-    } catch (BranchLayoutException e) {
-      val exceptionMessage = e.getMessage();
-      val errorMessage = "Error occurred while sliding out '${branchName}' branch" +
-          (exceptionMessage == null ? "" : ": " + exceptionMessage);
-      LOG.error(errorMessage);
-      VcsNotifier.getInstance(project).notifyError(/* displayId */ null,
-          getString("action.GitMachete.BaseSlideOutAction.notification.title.slide-out-fail.HTML").format(branchName),
-          exceptionMessage == null ? "" : exceptionMessage);
-    }
+    ApplicationManager.getApplication()
+        .invokeLater(() -> WriteAction
+            .run(() -> {
```

Comment:
Hmmm isn't there sth like `Application#runWriteAction` that would simplify that from 2 to 1 method call?

Reply:
"Must be called from the Swing dispatch thread" so won't suit us.

<img width="458" alt="image" src="https://user-images.githubusercontent.com/61919032/201661577-9a53dabf-f45d-4a30-a847-033d42ff630f.png">

Reply:
Awww okay... maybe add a helper method of our own in e.g. `com.virtuslab.gitmachete.frontend.actions.common.ActionUtils`